src/tools/schema2base.ts

```diff
@@ -29,18 +30,19 @@ export abstract class Schema2Base {
   }
 
   private async processFile(src: string) {
+    Utils.init('result: ${process.cwd()}');
```

Comment:
The first argument to `Utils.init` needs to be the relative path from CWD to arcs-root, or an absolute path to arcs-root (I tend to use the former because of point 3 below).

There are three problems here:
1. This argument will literally always have the value `'result: ${process.cwd()}'`. You have to use backticks (`` ` ``) to access interpolations (`${...}`). So at least `` `result: ${process.cwd()}` ``
2. Obviously, `result: ` is not part of a valid path
3. Even if you just use `Utils.init(process.cwd())` this will only work when `arcs` is at `\`, which it never is on my server (which may not be a blocker, since I'm an outlier on this point).

Reply:
Updated to be the same as the path call in Utils.parse.

Reply:
Sorry, this is different how? Seems like all three problems still exist ... ?